ctms/acoustic_sync.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+"""Run continuously in the background, syncing acoustic with our db."""
+
+from time import monotonic, sleep
+
+from ctms import config
+from ctms.database import get_db_engine
+
+
+def main(db, settings):
+    prev = monotonic()
```

Comment:
Nice, I need to remember to use ``time.monotonic()`` rather than ``time.time()`` for timing